src/shared/config.ts

```diff
@@ -56,6 +56,7 @@ export interface ServiceConfig {
     generateTimeout: number;
   };
   ENDPOINT_SERVICE_URL: string;
+  SERVICE_PROVIDER_URL: string;
```

Comment:
Hmm. I wonder if we can use a better term. While `SERVICE_PROVIDER` is correct, service provider means many different things. Maybe `PARTICIPANT_LIST_SERVICE_URL`?

Reply:
sure.